romanisim/util.py

```diff
@@ -194,7 +194,7 @@ def unflatten_value(k, v):
             if 'file' in k:
                 v = stnode.FileDate(v)
         except Exception as e:
-            pass
+            raise Exception(e)
```

Comment:
I think this change will actually have an effect.  I forget all of the different kinds of objects that could be in one of these metadata files, and which types of keys caused problems.  But I got tired of tracking them down and they weren't needed for the CRDS queries, so I ended up just catching all exceptions here and ignoring them.  The approach you once suggested re doing this through a image model metadata structure is likely a better approach, but let's not change this without actually fixing this here.

Reply:
I agree, if you are raising a general exception then you should get rid of the `try` block

Reply:
Let's just leave this as a blanket suppression of the possible exceptions here for now.  I agree this is bad practice, but the proposed change will break things and RDM-388 proposes a better way to handle this in general.  Or we can imagine a separate PR that does the translation between CRDS metadata and roman data model metadata in both directions.  But let's not mix that with this PR.

Reply:
Why not have it just turn the errors into warnings at the very least? That way you have an idea of what is going on.